var/spack/repos/builtin/packages/py-setuptools-scm/package.py

```diff
@@ -40,3 +40,5 @@ class PySetuptoolsScm(PythonPackage):
     depends_on("py-tomli@1:", when="@7:", type=("build", "run"))
     depends_on("py-typing-extensions", when="@7:", type=("build", "run"))
     depends_on("py-importlib-metadata", when="@7: ^python@:3.7", type=("build", "run"))
+
+    depends_on("git", type="build")
```

Comment:
```suggestion
    depends_on("git", type=("build", "run"))
```
`setuptools_scm.git` seems to use git at run-time too. Technically `setuptools_scm.hg` requires mercurial at run-time too. Not sure if it's worth adding a dependency on that or not.

Reply:
Me neither but if someone has problems it should be easy to find this PR and add a dependency.